types/dragino/lse01/uplink.js

```diff
@@ -0,0 +1,40 @@
+function hexToBytes(hex) {
```

Comment:
use bits?

Reply:
We don't have this function in the Bits library